src/test/java/org/openrewrite/java/logging/ParameterizedLoggingTest.java

```diff
@@ -65,6 +65,44 @@ static void method(Logger logger, String name) {
         );
     }
 
+    @Test
```

Comment:
We tend to link tests back to the issue if there is one, as that often gives more background.
```suggestion
    @Test
    @Issue("https://github.com/openrewrite/rewrite-logging-frameworks/issues/135")
```